### PR TITLE
<DatePicker /> - add readOnly prop and add examples to width prop

### PR DESCRIPTION
**Pull request by bamai**
https://jira.wixpress.com/browse/DSM-1658
DatePicker already supports this prop, I added documentation and tests.
I also added examples to width prop in storybook.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6871/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6871

File: packages/wix-style-react/src/DatePicker/docs/index.story.js [hunk 1]
```diff
@@ -146,6 +146,19 @@ export default {
             source: examples.sizes,
           }),
 
+          example({
+            title: 'Read only',
+            text: `set DatePicker's input to be readOnly`,
```
Comment (TalZa1810) [inline packages/wix-style-react/src/DatePicker/docs/index.story.js:151]:
```suggestion
            text: 'Sets the DatePicker input to be readOnly`,
```

File: packages/wix-style-react/src/DatePicker/docs/index.story.js [hunk 2]
```diff
@@ -146,6 +146,19 @@ export default {
...
+            source: examples.readOnly,
+          }),
+
+          example({
+            title: 'Width',
+            text:
+              '`<DatePicker />` supports changing its width. default width is 150px',
```
Comment (TalZa1810) [inline packages/wix-style-react/src/DatePicker/docs/index.story.js:158]:
```suggestion
              '`<DatePicker />` supports changing its width. Default width is 150px',
```

File: packages/wix-style-react/src/DatePicker/docs/index.story.js [hunk 3]
```diff
@@ -146,6 +146,19 @@ export default {
...
+            source: examples.width,
+          }),
+
```
Comment (TalZa1810) [inline packages/wix-style-react/src/DatePicker/docs/index.story.js:161]: @EgoziE Can we delete the commented example?

File: packages/wix-style-react/src/DatePicker/DatePicker.js [hunk 4]
```diff
@@ -378,4 +378,7 @@ DatePicker.propTypes = {
 
   /** Specifies the size of the input */
   size: PropTypes.oneOf(['small', 'medium', 'large']),
+
+  /** Sets the DatePicker input to readOnly */
```
Comment (TalZa1810) [inline packages/wix-style-react/src/DatePicker/DatePicker.js:382]:
```suggestion
  /** Sets the DatePicker input to be readOnly */
```

**Review by TalZa1810 — APPROVED** [review]
Few tiny comments. Looks good overall :)

**Comment by bamai** [comment]
@assafzi @sivanharel 
I changed here the default value of width prop, this is a breaking change and it might affect tpa

**Comment by assafzi** [comment]
UX approve.
Thanks!

**Pull request merged**